apispec/schema.yml

```diff
@@ -680,6 +768,45 @@ components:
               type: string
               format: uri
               example: /transactions/{transaction_id}/insolvency/statement-of-affairs
+
+    ProgressReportWritable:
+      type: object
+      properties:
+        progress_report_from_date:
```

Comment:
Given the context of the fields being in a progress report resource I think these date fields can be shortened to just "from_date" and "to_date". Same for the non-writable resource below too

Reply:
fixed